src/app/dev/platforms/desktop/DevToys.Linux/LinuxProgram.cs

```diff
@@ -31,6 +31,7 @@ internal partial class LinuxProgram
 
     internal LinuxProgram()
     {
+        Gio.Module.Initialize();
```

Comment:
You can remove this line. This is part of the sample as the Gio.Module is not initialized automatically. 

For Libadwaita and GTK applications this is done automatically via the `Gtk.Applocation` class for all GTK dependencies including Gio.

- [Gtk.Application](https://github.com/gircore/gir.core/blob/main/src/Libs/Gtk-4.0/Public/Application.cs) calls `Gtk.Module.Initialize`
- `Gtk.Module.Initialize` calls [Gdk.Module.Initialize](https://github.com/gircore/gir.core/blob/295288990b0f2972d0cbf898c937eea48f6109b8/src/Libs/Gtk-4.0/Public/Module.cs#L35)
- `Gdk.Module.Initialze` calls [Gio.Module.Initialize](https://github.com/gircore/gir.core/blob/295288990b0f2972d0cbf898c937eea48f6109b8/src/Libs/Gdk-4.0/Public/Module.cs#L38)